packages/pluggableWidgets/heatmap-chart-web/package.json

```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "@mendix/piw-utils-internal": "^1.0.0",
     "@mendix/shared-charts": "^1.0.0",
-    "classnames": "^2.2.6"
+    "classnames": "^2.2.6",
+    "date-fns": "^2.24.0"
```

Comment:
Didn't lock this yet cuz it will cause hoisting conflicts with date-grid. I think it's better if we fix it in a separate PR